### PR TITLE
fix: properly format crypto currencies

### PR DESCRIPTION
**Pull request by dated**
As of recently the currency mixins would add an additional whitespace between the currency symbol and the value when formatting crypto currencies (one regular space, one non-breaking space).

@luciorubeens this should be fine now. It removes the whitespace, then adds a non-breaking space instead of the regular space that was added before.

## What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactor
- [ ] Performance
- [ ] Tests
- [ ] Build
- [ ] Documentation
- [ ] Code style update
- [ ] Continuous Integration
- [ ] Other, please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Does this PR release a new version?

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `develop` branch, _not_ the `master` branch
- [x] All tests are passing
- [x] New/updated tests are included

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

### CODE REVIEW

**Comment by dated** [comment]
```
const randomPeers2 = store.getters['peer/randomPeers']()
expect(randomPeers).not.toEqual(randomPeers2)
```

this works until it doesn't :smirk:

File: __tests__/unit/mixins/currency.spec.js [hunk 1]
```diff
@@ -63,20 +63,21 @@ describe('Mixins > Currency', () => {
       })
     })
 
-    describe('when the `currencyFrom` option with value "network" is provided', () => {
+    describe.only('when the `currencyFrom` option with value "network" is provided', () => {
```
Comment (faustbrian) [inline __tests__/unit/mixins/currency.spec.js]: `only` :suspect:

Comment (dated) [reply __tests__/unit/mixins/currency.spec.js]: :hide:

**Comment by dated** [comment]
:trollface:

**Comment by ItsANameToo** [comment]
that doesn't seem right 🤔

**Pull request merged**